playgrounds/vite/src/app.tsx

```diff
@@ -1,6 +1,6 @@
 export function App() {
   return (
-    <div className="m-3 p-3 border">
+    <div className="m-3 p-3 border bg-red-500/mymy">
```

Comment:
Can remove these playground files from the diff 👍

Reply:
ah, my bad.